### PR TITLE
Update to support swagger

### LINKED ISSUE #7: Rest Client Panic
```
$ gaiacli rest-server --chain-id=chain-LldPl4 --home ./build/node0/gaiacli
panic: statik/fs: no zip data registered

goroutine 1 [running]:
main.registerSwaggerUI(0xc000a22dc0)
	/Users/aleksbez/go/src/github.com/cosmos/gaia/cmd/gaiacli/main.go:185 +0x14f
main.registerRoutes(0xc000a22dc0)
	/Users/aleksbez/go/src/github.com/cosmos/gaia/cmd/gaiacli/main.go:170 +0x43
github.com/cosmos/cosmos-sdk/client/lcd.ServeCommand.func1(0xc000aae000, 0xc0001c3f20, 0x0, 0x3, 0x0, 0x0)
	/Users/aleksbez/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.28.2-0.20190518230608-cfe31c409023/client/lcd/root.go:90 +0x4f
github.com/spf13/cobra.(*Command).execute(0xc000aae000, 0xc0001c3e60, 0x3, 0x3, 0xc000aae000, 0xc0001c3e60)
	/Users/aleksbez/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001b4c80, 0x47bd75f, 0xc0001b4c80, 0x4b7a365)
	/Users/aleksbez/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/aleksbez/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc0001b4c80, 0x4cf3e48, 0x2, 0xc0004e3f00)
	/Users/aleksbez/go/pkg/mod/github.com/tendermint/tendermint@v0.31.5/libs/cli/setup.go:89 +0x3c
main.main()
	/Users/aleksbez/go/src/github.com/cosmos/gaia/cmd/gaiacli/main.go:110 +0x8c8
```

### PR DESCRIPTION
**Pull request by alexanderbez**
closes: #7 

Move swagger client from SDK to gaia. Currently lives under `cmd/gaiacli/` now. Can be moved elsewhere?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,6 +103,9 @@ draw-deps:
 	go get github.com/RobotsAndPencils/goviz
 	@goviz -i ./cmd/gaiad -d 2 | dot -Tpng -o dependency-graph.png
 
+update_gaia_lite_docs:
```
Comment (alessio) [inline Makefile]: Very very minor thing: I tried to adopt `-` separated target naming convention for the new Gaia makefile. Not a big deal either way

**Review by alessio — APPROVED** [review]
Good one! - IMHO it shouldn't be moved elsewhere

File: Makefile (hunk 1 above)
Comment (alexanderbez) [reply Makefile]: Yes, copy-pasta and I agree!

**Review by sabau — APPROVED** [review]
Thanks! I was going to start this later :)

**Pull request merged**